SMPyBandits/Policies/RAWUCB.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import division, print_function  # Python 2 compatibility
+
+__author__ = "Julien Seznec"
+__version__ = "0.1"
+
+from math import sqrt, log
+import numpy as np
+import pandas as pd
+
+np.seterr(divide='ignore')  # XXX dangerous in general, controlled here!
+
+try:
+    from .BasePolicy import BasePolicy
+    from .IndexPolicy import IndexPolicy
+    from .FEWA import FEWA, EFF_FEWA
+    from .kullback import klucbBern
+except ImportError:
+    from BasePolicy import BasePolicy
+    from FEWA import FEWA, EFF_FEWA
+    from kullback import klucbBern
+    from IndexPolicy import IndexPolicy
+
+
+class EFF_RAWUCB(EFF_FEWA):
+    """
+    Efficient mechanism as described in [Seznec et al.,  2019a] (m=2) and [Seznec et al.,  2019b] (m<=2)
+    Algorithm Rotting Adaptive Window Upper Confidence Bound (RAW-UCB) [Seznec et al.,  2019b]
+    We use the confidence level \delta_t = \frac{1}{t^\alpha}.
```

Comment:
Same as above: add the full name and URL of the references, and reStructuredText markup around the maths